packages/react/src/Bento/Bento.features.stories.tsx

```diff
@@ -0,0 +1,163 @@
+export const EnterpriseTwo: StoryFn<typeof Bento> = () => (
+  <Bento>
+    <Bento.Item columnSpan={12} rowSpan={4} />
+    <Bento.Item columnSpan={5} rowSpan={4} />
+    <Bento.Item columnSpan={7} rowSpan={4} />
+    <Bento.Item columnSpan={12} rowSpan={4} />
+  </Bento>
+)
+
+export const EnterpriseThree: StoryFn<typeof Bento> = () => (
+  <Bento>
+    <Bento.Item columnSpan={7} rowSpan={4} />
+    <Bento.Item columnSpan={5} rowSpan={4} />
+    <Bento.Item columnSpan={12} rowSpan={4} />
+    <Bento.Item columnSpan={12} rowSpan={4} />
+  </Bento>
+)
+
+export const MixedThemeItems: StoryFn<typeof Bento> = () => (
+  <Bento>
+    <Bento.Item columnSpan={7} rowSpan={4} colorMode="dark" />
+    <Bento.Item columnSpan={5} rowSpan={4} />
+    <Bento.Item columnSpan={12} rowSpan={4} />
+    <Bento.Item columnSpan={12} rowSpan={4} colorMode="dark" />
+  </Bento>
+)
+
+export const ResponsiveItems: StoryFn<typeof Bento> = () => (
+  <Bento>
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+  </Bento>
+)
+
+export const ImageGallery: StoryFn<typeof Bento> = () => (
+  <Bento>
+    <Bento.Item columnSpan={7} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={5} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={12} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={12} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+  </Bento>
+)
+
+export const Example: StoryFn<typeof Bento> = () => (
+  <>
+    <style>
+      {`.purple-background {
+                background: linear-gradient(90deg, #6C5DD3 0%, #613DB4 100%);
```

Comment:
Could we use our base color scale values here instead of the hard coded values? Also the formatting seems off, could we move this into a separate stylesheet perhaps? E.g. Bento.features.stories.module.css